packages/transformer/src/test/standalone/GeoLocationTransform.test.ts

```diff
@@ -191,3 +213,139 @@ describe("Linear Geolocation Transformations", () => {
     transfrom.dispose();
   });
 });
+
+describe("Non Linear Geolocation Transformations", () => {
```

Comment:
Might be good to refactor the test so you only have to provide two sets of transforms data to a helper function, and that way you can validate many transforms without replicating the whole test over and over. 

Definitely a bunch of bad paths to test as well.

Reply:
We've discussed. Might not save any typing or time to extract, but go ahead and write some more cases and then maybe some helpers will emerge.